sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernatePoolDao.java

```diff
@@ -187,7 +187,8 @@ public SessionFactory getSessionFactory() {
   @Override
   public List<Pool> listByProjectId(long projectId) throws IOException {
     Criteria idCriteria = currentSession().createCriteria(PoolImpl.class, "p");
-    idCriteria.createAlias("p.pooledElementViews", "dilution");
+    idCriteria.createAlias("p.poolDilutions", "poolDilution");
+    idCriteria.createAlias("poolDilution.poolableElementView", "dilution");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "poolDilution.poolableElementView" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)